docker/lib/dependabot/docker/file_parser.rb

```diff
@@ -61,15 +64,18 @@ def parse
           end
         end
 
+        workflow_files.each do |file|
```

Comment:
Is "workflow_file" typically how you'd refer to these config files? If that's typical for folks in the community then 👍, I've heard "manifest" I think? https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-manifest

You obv have a much better idea on this, so curious to hear your thoughts

Reply:
What about `kubernetes_resource_template_files`?

Reply:
switched to manifests, this was a copy and paste left over.